auth-service/src/test/kotlin/uk/gov/communities/delta/controllers/AdminEnableDisableUserControllerTest.kt

```diff
@@ -243,16 +244,14 @@ class AdminEnableDisableUserControllerTest {
                         }
                     }
                     routing {
-                        bearerTokenRoutes(
-                            mockk(relaxed = true),
-                            mockk(relaxed = true),
-                            mockk(relaxed = true),
-                            mockk(relaxed = true),
-                            mockk(relaxed = true),
-                            mockk(relaxed = true),
-                            mockk(relaxed = true),
-                            controller,
-                        )
+                        withBearerTokenAuth {
+                            post("/bearer/admin/enable-user") {
```

Comment:
I imagine it makes no difference but since we are refactoring for consistency anyway I think in some places you have removed bearer from this bit and the urls in the test but in others you haven't? I think without looks tidier personally?

Reply:
Done, I agree it looks neater :)